assignments/Assignment5.js

```diff
@@ -0,0 +1,53 @@
+// Use nested loops to create diamond pattern
+
+function diamondPattern(centerWidth) {
```

Comment:
This function works just fine - I can see it generates the diamonds of the correct width.  It should though `return s;` as its final statement.  The caller can then receive the diamond patter back and either log it or put it into the HTML doc for display there.